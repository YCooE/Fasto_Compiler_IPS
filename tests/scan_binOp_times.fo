fun int write_int(int i) = write(i)

fun [int] write_int_arr([int] i) = map(write_int, i)

fun [int] main() =
    let x = {1, 2, 3, 4} in
    let r = scan(op *, 2, x)
    in write_int_arr(r)
