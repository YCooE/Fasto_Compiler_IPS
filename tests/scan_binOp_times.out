2 4 12 48
